server/routes/fixtures/supertest-assertions.ts

```diff
@@ -0,0 +1,69 @@
+import * as superagent from 'superagent'
+// eslint-disable-next-line import/no-extraneous-dependencies
+import cheerio, { CheerioAPI } from 'cheerio'
+
+export default function assertThat(response: superagent.Response) {
+  return new SupertestAssertions(response)
+}
+
+class SupertestAssertions {
```

Comment:
This is effectively our DSL for making assertions. All methods are chainable and is modelled heavily on assertJ (mainly because I couldnt work out how to write my own `expect` method that didnt conflict with jest's `expect`)

Also, in most assertion methods I have a try/catch with a `fail` (re: the change to package.json to bring in jasmine)
Its a bit long winded but its because jest's `expect` method doesnt support failure messages 😢 
Long winded? yes; but its nicely abstracted away into this class, so test methods dont need to worry about it